DOCS-2305: Email corrections (#1306)
MultiSafepay/docs: content/payment-methods/wallet/alipay/activate-alipay.md

## Activate 
>_Please check your eligibility with your MultiSafepay account manager before applying for this payment method._

> Alipay is an additional payment method. Therefore, an application is required. Submit your request by contacting your account manager or email our Integration Team at <integration@multisafepay.com>

Reviewer: ```suggestion
Alipay is an additional payment method. To apply, email your account manager at <sales@multisafepay.com> or the Integration Team at <integration@multisafepay.com>

```
